src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs

```diff
@@ -99,7 +99,7 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 			{
 				new ContextMenuFlyoutItemViewModel()
 				{
-					Text = "LayoutMode".GetLocalizedResource(),
```

Comment:
Can you do the same for the settings dialog and wherever else this string is used?

Reply:
Do you like to remove this resource?

Reply:
That would make sense

Reply:
Thanks!